addons/test_mail/models/test_mail_models.py

```diff
@@ -143,7 +143,18 @@ class MailTestStandard(models.Model):
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True)
     container_id = fields.Many2one('mail.test.container', tracking=True)
     company_id = fields.Many2one('res.company')
-
+    track_hidden_fields = fields.Char()  # comma-separated list
```

Comment:
Maybe we should be more test-oriented, like
```suggestion
    track_fields_tofilter = fields.Char()  # comma-separated list
```